python/samples/04.ai.b.messageExtensions.AI-ME/env/.env.local.user

```diff
@@ -1,4 +1,4 @@
 # This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
 
-# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
```

Comment:
Would be better to keep the comments so users can quickly understand the `SECRET_` prefix.